src/Extension/SiteConfigExtension.php

```diff
@@ -63,7 +69,32 @@ public function updateCMSFields(FieldList $fields)
         ));
         $mfaGraceEnd->addExtraClass('mfa-settings__grace-period');
 
-        $mfaOptions = CompositeField::create($mfaOptions, $mfaGraceEnd)
+        $mapFn = function ($groups = []) {
```

Comment:
Why are you declaring this as a function just to use it once? You can just put this inline to keep it simple.